app/src/Pages/Portal/Contributions/Utils/ContributionsFields.js

```diff
@@ -0,0 +1,351 @@
+    TextField,
+    Yup.string("Select your city"),
+    "Your city is required"
+  ),
+  state: requiredFormField(
+    "State",
+    FormSectionEnum.CONTRIBUTOR,
+    SelectField,
+    Yup.string("Select your state"),
+    "Your state is required",
+    stateList
+  ),
+  zipcode: requiredFormField(
+    "Zip Code",
+    FormSectionEnum.CONTRIBUTOR,
+    TextField,
+    Yup.number("Enter your zipcode"),
+    "A zipcode is required"
+  ),
+  contactType: formField(
+    "Contact Type",
+    FormSectionEnum.CONTRIBUTOR,
+    SelectField,
+    Yup.string("Select the best way to contact you"),
+    [
+      ContactTypeFieldEnum.MOBILE_PHONE,
+      ContactTypeFieldEnum.WORK_PHONE, 
+      ContactTypeFieldEnum.EXTENSION,
+      ContactTypeFieldEnum.HOME_PHONE,
+      ContactTypeFieldEnum.FAX,
+      ContactTypeFieldEnum.EMAIL
+    ]
+  ),
+  contactInformation: formField(
+    "Contact Information",
+    FormSectionEnum.CONTRIBUTOR,
+    TextField,
+    Yup.string("Enter your contact information")
+  ),
+  occupation: formField(
+    "Occupation",
+    FormSectionEnum.CONTRIBUTOR,
+    SelectField,
+    Yup.string("Select your occupation"),
+    ["Not Employed", "Self Employed", "Other"]
+  ),
+  employerName: formField(
+    "Employer's Name",
+    FormSectionEnum.CONTRIBUTOR,
+    TextField,
+    Yup.string("Enter your employer's name")
+  ),
+  employerCity: formField(
+    "Employer City",
+    FormSectionEnum.CONTRIBUTOR,
+    TextField,
+    Yup.string("Enter your employer's city")
+  ),
+  employerState: formField(
+    "Employer State",
+    FormSectionEnum.CONTRIBUTOR,
+    SelectField,
+    Yup.string("Enter your employer's state"),
+    stateList
+  ),
+  employerZipcode: formField(
+    "Employer Zip Code",
+    FormSectionEnum.CONTRIBUTOR,
+    TextField,
+    Yup.string("Enter your employer's zipcode")
+  ),
+
+  // OTHER DETAILS SECTION
+  electionAggregate: requiredFormField(
+    "Election Aggregate",
+    FormSectionEnum.OTHER_DETAILS,
+    TextField,
+    Yup.number("Enter your election aggregate"),
+    "Election aggregate is required"
+  ),
+  // REQUIRED IF: In-Kind Contribution, In-Kind Forgiven Accounts Payable,
+  // or In-Kind Forgiven Personal Expenditure was selection.
+  description: formField(
+    "Description",
+    FormSectionEnum.OTHER_DETAILS,
+    SelectField,
+    Yup.string("Choose the description of the contribution"),
+    [
+      "Broadcast Advertising",
+      "Fundraising Event Expenses",
+      "General Operating Expenses",
+      "Literature/Brochures/Printing",
+      "Management Services",
+      "Newspaper and Other Periodical Advertising",
+      "Other Advertising",
+      "Petition Circulators",
+      "Postage",
+      "Preparation and Production of Advertising",
+      "Surveys and Polls",
+      "Travel Expenses",
+      "Utilities",
+      "Wages/Salaries/Benefits"
+    ]
+  ),
+  // Not required if occupation & employer name/address filled in
+  occupationLetterDate: formField(
```

Comment:
Not sure if this needs to be a date. Drop down shown on figma.

Reply:
I was going off the spec here for that field: https://docs.google.com/document/d/1djgOlGz1RJxItbmbSphsmS4oyTPRSd0VlzLgBNDlIYo/edit
This might be out of date though!